src/Tickets/Admin/Onboarding/Tickets_Landing_Page.php

```diff
@@ -552,7 +552,7 @@ public function admin_content_checklist_section(): void {
 							<?php esc_html_e( 'Stripe for online payments', 'event-tickets' ); ?>
 						</div>
 						<div class="step-list__item-right">
-							<a href="<?php echo esc_url( admin_url( "{$settings_url}&tc-section=stripe&tab=stripe" ) ); ?>" class="tec-admin-page__link">
+							<a href="<?php echo esc_url( admin_url( "{$settings_url}&tab=tickets-commerce" ) ); ?>" class="tec-admin-page__link">
```

Comment:
So the tab was incorrect...
The correct url _should_ be:
`wp-admin/admin.php?page=tec-tickets-settings&tc-section=stripe&tab=payments`

That sends them directly to the Stripe section of the payments tab